styles/newsletter.scss

```diff
@@ -49,16 +49,12 @@ $o-grid-gutter-small: oGridGutter(S);
 }
 
 .n-newsletter-signup__heading__title {
-	@include oTypographyDisplayBold($scale: 2);
-	font-size: 30px;
-	line-height: 30px;
+	@include oTypographySans($scale: 2);
```

Comment:
Could we do this using the responsive scale mixins? 
`@include oTypographySans( $scale: (default: (2, 1em), M: (0, 1em), XL: (2, 1em)) );`

Reply:
https://github.com/Financial-Times/n-newsletter-signup/pull/20/files#diff-ad04c49b4c5a5b160ad3fdc6610b9dfaR39